.teamcity/scripts/check_modified_rust_modules.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+CHANGED_FILES="%system.teamcity.build.changedFiles.file%"
+
+# Check if any files match your path pattern
+if echo "$CHANGED_FILES" | grep -q "\-rs"; then
+    echo "Changes detected for files with '-rs' in their path"
+    export KTOR_RUST_COMPILATION="true"
+    echo "##teamcity[setParameter name='env.KTOR_RUST_COMPILATION' value='${'$'}KTOR_RUST_COMPILATION']"
```

Shellcheck (0.10.0)

[error] Parameter expansion starts with unexpected quotes. Double check syntax.

(SC2301)